app/background.cc

```diff
@@ -18,16 +18,17 @@ struct background_t {
     int frm;
     double fps;
     cv::Mat raw;
+    cv::Mat thm;
```

Comment:
Hard disks are large enough for good identifiers (or comments explaining their purpose). ;-)

Reply:
I did wonder why I'd done that earlier - proper names coming up!